doc/classes/ConeTwistJoint.xml

```diff
@@ -30,28 +32,39 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_param" getter="get_param">
+			The strength with which the correction of an overstepped swing limit will take place, the higher, the stronger.
 		</member>
 		<member name="relaxation" type="float" setter="set_param" getter="get_param">
+			A factor that gets applied to the speed of all twist and swing impulses.
 		</member>
 		<member name="softness" type="float" setter="set_param" getter="get_param">
+			The ease with which the joint starts to twist, if it's too low, it takes more force to start twisting the joint.
 		</member>
 		<member name="swing_span" type="float" setter="_set_swing_span" getter="_get_swing_span">
+			The higher, the more swing (rotation around axes orthogonal to the twist axis) is needed to have the swing corrected. Disabled when under 0.05.
```

Comment:
Avoid parens. Avoid passive voice. Favor action verbs and direct voice. Example for the form (check it's correct, a lot is guesswork, I'm not familiar with the physics engine and joints in godot):
```Controls the frequency of swing corrections by the physics engine.
Swing is rotation from side to side, around the axis perpendicular to the twist axis. The lower the value, the more often the physics engine will smooth out the swing. 
A value lower than [code]0.05[/code] disables all swing corrections. Default value: [code]value[/code]```

Reply:
thanks, relatively precisely what i think it does, i should definitely not be so shy to write multiple lines of explanation :)